README.md

```diff
@@ -1,2 +1,12 @@
 # comp110-worksheet-B
```

Comment:
Flowchart is a little minimal and lacking in detail. Would be clearer to split the checks for correct guesses and too many guesses into separate decision nodes. Also it would be clearer to have an explicit counter for guesses that is incremented on each incorrect guess.